test/namedtuple.jl

```diff
@@ -95,6 +95,14 @@ end
 @test !isless((a=2,b=1), (a=1,b=2))
 @test_throws MethodError isless((a=1,), (x=2,))
 
+@test (a=1,b=2) < (a=1,b=3)
+@test_broken (a=1,) < (a=1,b=2)
```

Comment:
This should probably throw an error? Of course that's not possible until 2.0, but maybe a deprecation should be printed?

Reply:
This is an error. I thought the `@test_broken` above was intentional. I will change this.

Reply:
The relevant TODO is https://github.com/JuliaLang/julia/pull/40147/files#diff-8bd5cf0ce6c0ef236a5ae91851437fc8197d8f56c70b2b5dedd73ef55bb18e17L198, but personally I agree we should not accept prefixes.